exes/mqtt-broadcaster/inc/mqtt_client.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <openssl/ssl.h>
+#include <async_mqtt/buffer.hpp>
+#include <async_mqtt/endpoint.hpp>
+#include <async_mqtt/packet/qos.hpp>
+#include <async_mqtt/packet/v5_publish.hpp>
+#include <async_mqtt/predefined_underlying_layer.hpp>
+#include <async_mqtt/tls.hpp>
+#include <boost/asio/any_io_executor.hpp>
+#include <boost/asio/awaitable.hpp>
+#include <boost/asio/co_spawn.hpp>
+#include <boost/asio/connect.hpp>
+#include <boost/asio/detached.hpp>
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/ip/tcp.hpp>
+#include <boost/asio/strand.hpp>
+#include <boost/asio/use_awaitable.hpp>
+
+#include <sparkplug_b/sparkplug_b.pb.h>
+#include <config/mqtt_config.hpp>
+#include <tfc/confman.hpp>
+#include "constants.hpp"
+#include "impl.hpp"
+
+namespace tfc::mqtt {
+
+using org::eclipse::tahu::protobuf::Payload;
+using org::eclipse::tahu::protobuf::Payload_Metric;
+
+namespace asio = boost::asio;
+
+class client {
+public:
+  explicit client(asio::io_context& io_ctx, std::string will_topic);
```

Comment:
```suggestion
  explicit client(asio::io_context& io_ctx, std::string_view will_topic);
```
`will_topic` is not really descriptive, I don't know what it is

Reply:
changing name to `mqtt_will_topic`